tests/reversetest.ark

```diff
@@ -0,0 +1,11 @@
+{
+(import "Reverse.ark")
+    # --------------------------
+    #           Reverse
+    # --------------------------
+    (let reverse-tests (fun () {
+        (assert (= "CBA" (reverse "ABC")) "Slice test 1 failed")
+        (assert (= "Wow, it work" (reverse (reverse "Wow, it work"))) "Slice test 2 failed")
```

Comment:
just being an english teacher here, but that would be better to avoid having mistakes here, it's "work**s**" (because it's the 3rd person)

Reply:
also you wrote "slice test" but those are reverse tests